test/manifold_test.cpp

```diff
@@ -160,6 +160,31 @@ TEST(Manifold, Decompose) {
   RelatedGL(manifolds, input);
 }
 
+TEST(Manifold, DecomposeProps) {
+  auto posNorm = [](float* newProp, glm::vec3 pos, const float* oldProp) {
+    newProp[0] = hypotf(hypotf(pos.x, pos.y), pos.z);
```

Comment:
`hypotf` is a new one for me, but I think it's simpler to go with `glm::length(pos)`.

Reply:
Probably calls the same method in the end!